### PR TITLE
[Error page] 404, etc.

### LINKED ISSUE #42: [Error page] 404 etc
We currently have no error page and non-existent urls result in a blank page.

We should add an error route [see like here](https://github.com/patocallaghan/realworld-starter-kit/blob/master/app/router.js#L23) and an error template [like here](https://github.com/patocallaghan/realworld-starter-kit/blob/master/app/templates/error.hbs) (yes it looks awful)

Want to write a test?
* Acceptance: nonsensical URL results in error page showing

### PR DESCRIPTION
**Pull request by delusioninabox**
Fixes #42 

- [x] Create route for error page
- [x] Direct to error page in router if path not found
- [x] Passing test to confirm redirect

### CODE REVIEW

File: app/routes/error.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Route from '@ember/routing/route';
```
Comment (Alonski) [inline app/routes/error.js]: Do we need this file if its empty?
If I remember correctly Ember will generate an empty route anyways no?

File: app/templates/error.hbs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div data-test-error-page class="editor-page">
+  <div class="container page">
+    <div class="row">
+      <div class="col-md-10 offset-md-1 col-xs-12">
+        <h2>Sorry but that page does not exist.</h2>
+        <p>Try find your page from the {{#link-to "home"}}homepage{{/link-to}}</p>
```
Comment (Alonski) [inline app/templates/error.hbs]: Maybe should be:
"Try `to` find your page from the homepage"

File: tests/acceptance/error-test.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import { visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Acceptance | Error', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  test('visiting /error', async function(assert) {
+    await visit('/some-BODY-once-told-me');
```
Comment (Alonski) [inline tests/acceptance/error-test.js]: Nice 🤣

File: app/routes/error.js (hunk 1 above)
Comment (patocallaghan) [reply app/routes/error.js]: Yep this can be 🔪 🎉

File: app/templates/error.hbs (hunk 2 above)
Comment (patocallaghan) [reply app/templates/error.hbs]: It reads fine to me 🙈 but then again I'm not sure if your suggestion is grammatically correct? 😄

Comment (Alonski) [reply app/templates/error.hbs]: I am not sure if "Try find" makes sense 😅

**Comment by delusioninabox** [comment]
Updated! 👍

**Pull request merged**